### PR TITLE
Bug 2193109: fixed startAndEndsWithAlphanumerics regex

### PR DESCRIPTION
**Pull request by ivoalmeida**
fixes https://bugzilla.redhat.com/show_bug.cgi?id=2193109

### CODE REVIEW

File: packages/odf/utils/validation.ts [hunk 1]
```diff
@@ -2,7 +2,7 @@
  * Starts and ends with a lowercase letter or number
  */
 export const startAndEndsWithAlphanumerics =
-  /^(?![-.])([a-z0-9]|[\W]*([a-z0-9]))+(?![-.])$/;
+  /^(?![\W])([a-z0-9]|[\W]*([a-z0-9]))+(?![\W])$/;
```
Comment (SanjalKatiyar) [inline packages/odf/utils/validation.ts:5]: this will fail if any capital letter is present in the text ex: `jdjAd`
is this expected ??

Comment (SanjalKatiyar) [reply packages/odf/utils/validation.ts:5]: cc @ivoalmeida

Comment (ivoalmeida) [reply packages/odf/utils/validation.ts:5]: > this will fail if any capital letter is present in the text ex: `jdjAd` is this expected ??
yes

image

**Comment by SanjalKatiyar** [comment]
@ivoalmeida can u rebase this PR with latest master...

**Pull request merged**